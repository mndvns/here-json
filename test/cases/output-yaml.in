./bin/here-json -y <<EOF
---
  this: is
  some: yaml-y output
EOF