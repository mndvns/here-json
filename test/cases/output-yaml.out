---
  this: "is"
  some: "yaml-y output"